src/PaymentElement.res

```diff
@@ -369,13 +370,20 @@ let make = (
     </ErrorBoundary>
   }
 
-  let paymentLabel = showFields
-    ? localeString.selectPaymentMethodLabel
-    : localeString.savedPaymentMethodsLabel
+  React.useEffect0(() => {
```

Comment:
Should there be a dependency here on paymentMethods?
cc: @prafulkoppalkar @vsrivatsa-juspay

Reply:
Updated